TracksDemo/TracksDemo.xcodeproj/project.pbxproj

```diff
@@ -708,14 +765,18 @@
 			baseConfigurationReference = 0262514F2B990CA2F3D99C07 /* Pods-TracksDemo.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				GCC_PREFIX_HEADER = TracksDemo/TracksDemo_Prefix.pch;
 				INFOPLIST_FILE = TracksDemo/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
The `Podfile` for the demo project sets 12.0 as the deployment target. We can't use that here because of SwiftUI, but should we make them meet in the middle at 13.0?

I can see an argument for running on the latest iOS only and not have to worry about backward compatibility. At the same time, Tracks is not restricted to iOS 14.0 only, so it might be beneficial for the demo app to run against as many compatible versions of iOS as reasonable - that is, iOS 13 so we can keep test the SwiftUI-related stuff.

Reply:
Yeah seems reasonable – changed to iOS 13 in 9ed5b7a08e4ec26b97358489e1741d2cd5451c7c